docs/dymos_book/api/phase_api.ipynb

```diff
@@ -220,12 +220,6 @@
     "        :noindex:\n",
     "```\n",
     "\n",
-    "## set_polynomial_control_options\n",
```

Comment:
Let's leave this in as deprecated.  Also search the `ipynb` files in docs and make sure that we use the new API in all of the documented examples.

Reply:
Got it, I've added that back in and searched through the docs and switched to using the new API; checking CI results now.